src/panels/lovelace/editor/config-elements/hui-calendar-card-editor.ts

```diff
@@ -112,20 +114,29 @@ export class HuiCalendarCardEditor extends LitElement
           @value-changed=${this._valueChanged}
         ></hui-theme-select-editor>
       </div>
-      <h3>
-        ${this.hass.localize(
-          "ui.panel.lovelace.editor.card.calendar.calendar_entities"
-        ) +
-        " (" +
-        this.hass!.localize("ui.panel.lovelace.editor.card.config.required") +
-        ")"}
-      </h3>
-      <ha-entities-picker
-        .hass=${this.hass!}
-        .value=${this._configEntities}
-        .includeDomains=${["calendar"]}
-        @value-changed=${this._valueChanged}
-      >
+      ${
+        this.editorOptions?.hide_entities
```

Comment:
I find this a very invasive change. Changing every editor current and future for just 1 card. And it also means that we can't optimize the UI without having to consider how it looks in the filter editor UI. 

I think that we're better off not merging this. It will hurt us too much in the future.

Reply:
I see two alternative ways to handle this.

First option is my first solution, where `entities:` is given a magic value, which makes the entity picker not display at all or show a message about entities being selected from elsewhere. The responsibility is then offloaded entirely by `hui-entity-filter-card-editor` and `ha-entities-picker`.

The middle path would be to change the interface of `ha-entites-picker` and have any card that uses it forward the entire `editorOptions` and let it do its thing or not. That gives some responsibility to the individual card editors, but much less than the solution in this PR.

I should give it some more thought, but I like the second alternative...

Reply:
A similar approach (expanding `editorOptions`) could probably be used to implement the functionality mentioned here https://github.com/home-assistant/frontend/pull/7637#discussion_r525154328

Reply:
I am afraid that we're going to code ourselves into a corner trying to support this. 

If we ever want to overhaul Lovelace editors with an improved UI, this change will hold us back.